tests/integration/test_end_to_end.py

```diff
@@ -18,7 +18,7 @@
 _VALID_NHSD_TARGET_IDENTIFIER_BASE_64 = base64.b64encode(bytes(json.dumps(_VALID_NHSD_TARGET_IDENTIFIER), 'utf-8'))
 
 _EXPECTED_CORRELATION_ID = "123123-123123-123123-123123"
-_MATCH_PATIENT_NHS_NUMBER = "9912003888"
+_MATCH_PATIENT_NHS_NUMBER = "9900000285"
```

Comment:
Could this be swapped to utilise the NHS number fixture?

Reply:
Its used in a '@pytest.mark.parametrize' which doesn't/is very hard to make use fixtures - I expect that is why it was specified as a global here